src/responder/common/responder_common.c

```diff
@@ -203,6 +202,17 @@ errno_t csv_string_to_uid_array(TALLOC_CTX *mem_ctx, const char *csv_string,
     int list_size;
     uid_t *uids = NULL;
     char *endptr;
+    const char *envvar;
+    bool loops_were_allowed;
+
+    envvar = getenv("_SSS_LOOPS");
+    loops_were_allowed = (envvar == NULL || strcmp(envvar, "NO") != 0);
+
+    ret = setenv("_SSS_LOOPS", "YES" , 1);
```

Comment:
I'm not sure why but existing pattern is to `unsetenv` instead of `setenv("YES")`.

This could be also wrapped into `if (!loops_were_allowed)` but perhaps not worth it.

Reply:
Done.